pkg/apis/serving/v1beta1/inference_service_validation.go

```diff
@@ -116,6 +117,10 @@ func validateInferenceService(isvc *InferenceService) (admission.Warnings, error
 		return allWarnings, err
 	}
 
+	if err := validateMultiNodeVariables(isvc); err != nil {
```

Comment:
For validations, take into consideration my comment in the other PR: https://github.com/kserve/kserve/pull/3871/files#r1774194794

Reply:
yes. I replied it in other thread. let's discuss this in that comment.

Reply:
added.